docs/index.md

```diff
@@ -62,13 +62,23 @@ TAU Profiles (mobile, TV, wearable) have differences, so some of the examples wi
 ## Where to get it
 
 The framework is included on all Tizen devices and the SDK emulator (starting from Tizen 2.3). Just create
-a new web application in the Tizen SDK IDE.
+a new web application in the Tizen SDK IDE. See also our [github repository](https://github.com/Samsung/TAU).
 
 ## What next?
 
```

Comment:
Please add link to TAU API documentation too
https://developer.tizen.org/development/api-references/web-application

Reply:
Done